### ISSUE TITLE
Print configuration in output for install/create

### ISSUE THREAD

**Issue by memsharded** [conan-io/conan#3458]
I find it a bit tricky to debug and find the used settings, specially in traces of CI. And tracing back to a profile file, might still fail to see if some setting or option was overwritten in the command line.

Maybe it would be useful to print it to output for these commands.

**Comment by lasote** [comment]
Do you suggest to print the "applied" profile like the trace of cpt?

```
+------------------+
| Profile          |
|------------------|
| include(default) |
|                  |
| [settings]       |
|                  |
| [options]        |
|                  |
| [env]            |
|                  |
| [build_requires] |
+------------------+
```

**Comment by memsharded** [comment]
Not the input profile, with ``include(default)``, but the evaluated profile. Which could be defined for local commands from the installed ``conaninfo.txt``.

**Comment by lasote** [comment]
That what I meant with "applied". Ok.

**Comment by sunj1** [comment]
Hi @memsharded ,

Thanks for the useful addition.
Could we also display values in `default_options` of the conanfile in this output?

**Comment by memsharded** [comment]
Hi @sunj1 

I am afraid it is not possible, because this is printed before the conanfile is loaded. Also, the default_options might be totally useless, if the profile is already defining those options, as they will be basically overriden when the conanfile is loaded, with the profile ones.

Maybe you want to use the new ``conan inspect . -a=default_options`` command for this purpose?

I wouldn't oppose to print further information of the conanfile being installed or created, what do yo think @lasote?

**Comment by sunj1** [comment]
If I understand it correctly, the objective of this was to know which settings and options are being used for the build.
If the entire set of options and settings are available after loading the conanfile (and before starting the build), wouldn't it be good to display the exact and all settings and options for the concerned package?

**Comment by memsharded** [comment]
Not really. The conanfile itself could be a header-only library, that doesn't declare settings. But its dependencies do, so the settings necessary are those of the profile, not the settings applied to the "base/root" conanfile. Same for options, there could be options declared in the profile, that only affect dependencies, and they won't be reflected in that consumer conanfile data.

This is why I think it makes sense to keep this output, as-is, and if anything, to add further output regarding the consumer conanfile.

**Comment by sunj1** [comment]
I think I would be interested in final options and settings applicable to only my consumer conanfile, and not those coming the dependencies.
In what ways can I print this information just before the build starts?

**Comment by memsharded** [comment]
You could run a ``conan inspect`` over the thing you will be building, to get the basic attributes printed.
If you want it to be automatic in the ``conan install`` or ``conan create`` commands, then I think you should open a new issue to request it, and it should be discussed. Thanks!
